docs/cloudwatch-alarms.md

```diff
@@ -0,0 +1,194 @@
+---
+id: cloudwatch-alarms
+title: CloudWatch Alarms
+---
+
+import TextToSpeech from '../src/SpeechComponent.js';
+
+<TextToSpeech>
+# CloudWatch Alarms
+
+# Introduction
+
+
+## What is CloudWatch?
+
+According to AWS,
+
+
+
+    Amazon CloudWatch is a monitoring and observability service. … (It) provides you with data and actionable insights to monitor your applications, respond to system-wide performance changes, and optimise resource utilisation.
```

Comment:
Looks like this is incomplete, why is the ... at the beginning of the second sentence